src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -8,7 +8,19 @@ tags:
   - name: Groups
 
 info:
-  description: Open Build Service API
+  description: |
+    The _Open Build Service API_ is a XML API.
+    This documentation is based on the _OpenAPI 3.0.0_ specification and powered by _Swagger_.
+
+    To authenticate, use [HTTP basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) by passing the _Authorization_ header in the form of `Authorization: Basic <credentials>`.
+
+    We don't have any API versioning as we don't have a need for it right now.
+
+    We don't have rate limiting, so be conscious about how you use the API.
```

Comment:
```suggestion
    We only implement rudimentary rate limiting, so please be gentle when using the API concurrently, especially with potentially expensive operations. In case of abuse we will limit/remove your access.
```

Reply:
I guess _rudimentary_ is the right word here... I like the second part of your suggestion.